README.md

```diff
@@ -131,7 +149,9 @@ Import SBOM files from either standard input or the local file system.
 bomctl import [flags] { - | FILE...}
```

Comment:
I missed that one 😅. Nice catch

Reply:
With the latest changes, the resulting merged document will always inherit all tags from the source documents. The `alias` option will always set the alias for the merged document and the `tag` option will always add tags to the merged document, so it will have the inherited and added tags.